VERSION

```diff
@@ -1,4 +1,4 @@
-$PG_VERSION  ='2.11';
+$PG_VERSION  ='develop';
 $PG_COPYRIGHT_YEARS = '1996-2016';
```

Comment:
I dont know why this is here since VERSION in the develop branch already says develop.  Its not causing conflicts though so it may not matter.  Merging develop into develop_translator_upd may fix it.